webapp/src/game/game.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity, Column, PrimaryGeneratedColumn } from "typeorm";
+import { Board } from "@kamon/core/src/board/board";
+import { GameState } from "@kamon/core/src/game/state";
+
+@Entity()
+export class Game {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column("json")
+  board: Board;
```

Comment:
I suggest to add players ids as two columns. And use foreign keys.

Reply:
It was also what I planned to do,  but it's still heavily WIP since I needed to fix migrations.